acceptance/scripts/generic/testrun-full.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -x
+
+# static fixtures
+REDIS_HOSTNAME=redis.delivery.puppetlabs.net
```

Comment:
This one: https://github.com/puppetlabs/pe-rbac-service/blob/master/integration/scripts/generic/testrun-full.sh#L9-L12